### DISCUSSION TITLE
Database docs

### DISCUSSION THREAD

**Discussion by carstenjaksch** [laravel/framework#37209] [Ideas]
While I was writing issue #37208, I thought to make an extra discussion for this.

First: Would be cool to see [official support](https://laravel.com/docs/8.x/database#introduction) for MariaDB, since Laravel Sail offers it during installation.

And maybe add a note about supported databases to each [column type](https://laravel.com/docs/8.x/migrations#available-column-types) in the docs and write down their fallbacks.
For example, `jsonb` is supported in PostgreSQL and falls back to `json` in MySQL.

Thoughts?

🖖

**Comment by shaedrich** [comment]
Additionally, it would be nice to have a list of methods where Laravel just silently does nothing without informing the user when they have a database that doesn't support this action natively.

**Reply by hafezdivandari** [reply]
Like `dropPrimary` on SQLite.
